src/ray/core_worker/object_interface.cc

```diff
@@ -11,13 +11,17 @@ namespace ray {
 void ObjectIdsByStoreProvider(
     const std::vector<ObjectID> &object_ids,
     EnumUnorderedMap<StoreProviderType, std::unordered_set<ObjectID>> *results) {
+  // There can be a few cases here:
+  // - for task return objects, determine the store provider type for an object
```

Comment:
This comment isn't really reflecting the current behavior (and therefore code structure). We only have a single special case (return object and direct actor object). Please update the comment to say that (1) for this one special case we use memory store provider and (2) all other cases use plasma.

Reply:
makes sense. that better describes what the code currently does.